etc/base-secrets.yaml

```diff
@@ -147,9 +147,9 @@ radar_integration:
 
 # --------------------------------------------------------- 20-s3-connector.yaml ---------------------------------------------------------
 # The access keys and secret keys of object storage services should match.
-# If AWS S3 is used as a storage medium instead of minio, then fill in those.
```

Comment:
I think we should keep the original comment, but perhaps clarify with:

```
# If AWS S3 is used as a storage medium instead of minio, then enter these secrets here.
```